src/plugins/home/public/application/components/sample_data_set_cards.js

```diff
@@ -81,7 +81,10 @@ export class SampleDataSetCards extends React.Component {
   loadSampleDataSets = async (dataSourceId) => {
```

Comment:
It seems we are adding workspaceId into the parameters of listSampleDataSets/install/uninstall but not in loadSampleDataSets, any reason for such difference?

Reply:
The `loadSampleDataSets` function call `listSampleDataSets` below. It will read the current workspace id from the workspace service directly. So we don't add the parameter for workspace id.